schemas/lockfile.schema.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://github.com/denoland/vscode_deno/blob/main/schemas/lockfile.schema.json",
+  "title": "Deno Lock file",
+  "description": "A lockfile that stores the hashes of all dependencies in a project to ensure integrity.\n\nSee https://deno.land/manual@v1.28.2/basics/modules/integrity_checking in the Deno documentation for more information.",
+  "type": "object",
+  "required": [
+    "version",
+    "remote"
+  ],
+  "additionalProperties": {
+    "type": "string",
+    "description": "The hash as the value for the remote URL as the key."
+  },
+  "properties": {
+    "version": {
+      "description": "The version of the lockfile.",
+      "type": "string",
+      "enum": [
+        "2"
+      ]
+    },
+    "remote": {
+      "description": "A map of remote URLs to their respective hashes.",
+      "type": "object",
+      "additionalProperties": {
+        "type": "string",
+        "description": "The hash as the value for the remote URL as the key."
+      }
+    }
```

Comment:
It's missing `npm` property that is used to store integrities of `npm` packages. You can see example lock file here: https://github.com/denoland/deno/blob/7b212bc5741ea0cfb6bd87c280be0c864cb32813/cli/tests/testdata/npm/lock_file/lock.json